### PR TITLE
added entry for Aslam Doctor

### PR DESCRIPTION
**Pull request by aslamdoctor**

### CODE REVIEW

**Comment by aurooba** [comment]
@aslamdoctor As noted in the in the [Contributing Guidelines](https://github.com/aurooba/sponsor-wp-contributors/blob/main/CONTRIBUTING.md), the list is alphabetized by *first name*. Could you please adjust your PR accordingly?

I'm also seeing changes to spacing, was there a reason you formatted the entire list differently?

**Comment by aslamdoctor** [comment]
Let me know if this last commit works for you.  Otherwise reject this PR :)

File: README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ Let's invest in the folks who create the software that empowers us to create a l
   - WordPress: [@afragen](https://profiles.wordpress.org/afragen/)
   - Sponsor: [GitHub Sponsor Program](https://github.com/sponsors/afragen)
 
+[Aslam Doctor](https://www.aslamdoctor.com)
+
```
Comment (aurooba) [inline README.md]: @aslamdoctor let's remove this space, the rest is perfect!

**Review by aurooba — CHANGES_REQUESTED** [review]
There's an extra space that needs to be removed under your name, once that's fixed up, we'll be all set. Thanks so much, Aslam!

File: README.md (hunk 1 above)
Comment (aslamdoctor) [reply README.md]: Thanks for review, the space has been removed

**Review by aurooba — APPROVED** [review]

**Comment by aurooba** [comment]
aaand merged! :) 🎉

**Comment by aslamdoctor** [comment]
Thank you 😊

**Pull request merged**